envoy.yaml

```diff
@@ -0,0 +1,68 @@
+static_resources:
```

Comment:
Are this file and the run target in Makefile still in used? Maybe we can remove them.

Reply:
I think maybe we should keep it ？Because someone may want to run it locally and they will need that file.

Reply:
It is OK to keep it for now.
In the future, we may support generating dummy yaml in the test framework so people can verify it locally.

Reply:
I got it.